Wikipedia/Code/DescriptionWelcomeInitialViewController.swift

```diff
@@ -32,15 +32,7 @@ class DescriptionWelcomeInitialViewController: UIViewController, Themeable {
     }
     
     override var supportedInterfaceOrientations : UIInterfaceOrientationMask {
```

Comment:
I think this property can be removed entirely now

Reply:
removed